java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -743,6 +781,7 @@ public ColumnVector findAndReplaceAll(ColumnVector oldValues, ColumnVector newVa
    */
   public ColumnVector isNull() {
     ColumnVector res = null;
+    // No prediction is possible because it is built up from other operators
```

Comment:
I see the comment in `isNotNull` was updated, I assume the same needs to be done here.

Reply:
I thought your issue was that the comment was not obvious.  In this one isNull is built up from other operators.  `validityAsBooleanVector` and the `not`.  If it is still confusing I can expand the comment out.

Reply:
My issue is with the `No prediction is possible` part -- it implies that this operation can never make a prediction.  How about something like:
```suggestion
    // Memory prediction handled by underlying operations
```